package.json

```diff
@@ -18,8 +18,7 @@
     "lodash": "^4.17.15",
     "pkginfo": "^0.4.1",
```

Comment:
Can this go away?

Reply:
We're using it, but not much. We could use the `exports.version = require('/path/to/package.json').version;` method that pkginfo decries.